### PR TITLE
[Modulo 3 lab sklearn and unsupervised learning] Alberto Pinon

### PR DESCRIPTION
**Pull request by albertopformoso**

### CODE REVIEW

**Comment by ta-data-mex** [comment]
Hola Alberto!
Cuando te dice "Inspect the first 5 records in wine['data']" se refiere a: wine['data'][:5]
Para el label encoder podrías hacer directamente para cada columna: 
patients["patient_diabetic"] = LabelEncoder().fit_transform(patients["patient_diabetic"])  *Porque sé que te gusta el rollo one liner 😉  
Igual para la columna labels, no necesitas hacer una lista: patients["labels"] = patients_cluster.labels_ 
Me parece que cuando haces el scatter plot y llamas a la columna 'labels' no te lo grafica bien por la manera en la que creaste la columna (corrí esa misma línea en mi código y sí lo hace bien).
El plot 3D lo puedes hacer así:
ax.scatter(patients["patient_height_sm"],patients["patient_weight_kg"],patients["patient_diabetic"],c=patients.labels); El output sería un poco diferente. 
Buen trabajo!!! ⭐

**Comment by albertopformoso** [comment]
Gracias, ya revisé y asigne mal las etiquetas y por eso no podía realizar
la gráfica con el puro dataframe... pero ya lo arregle.
